loinc-integration/src/test/java/dev/ikm/tinkar/loinc/integration/LoincAxiomSemanticIT.java

```diff
@@ -40,14 +71,59 @@ public void testLoincAxiomSemantics() throws IOException {
 
         assertEquals(0, notFound, "Unable to find " + notFound + " Loinc.csv 'Axiom' semantics. Details written to " + errorFile);
     }
-
+    
+    @Override
+    protected boolean assertLinePart(String[] columns) {
+        UUID id = uuid(columns[0]);
+        
+		Map<UUID, ConceptMapValue> termConceptMap = new HashMap<>(); 
+        
+		EntityProxy.Concept concept = EntityProxy.Concept.make(PublicIds.of(id));
+
+		String partNumber = removeQuotes(columns[0]);
+		String partTypeName = removeQuotes(columns[1]);
+		String partName = removeQuotes(columns[2]);
+	
+		LoincUtility.addPartToCache(partName.toLowerCase(), partTypeName, partNumber);
+
+		UUID uuidAxiom = uuid((concept.publicId().asUuidArray()[0] + partTypeName + "AXIOM"));
+	
+		final StateSet active;
+		if (columns[4].equals("ACTIVE") || columns[4].equals("TRIAL") || columns[4].equals("DISCOURAGED")) {
+			active = StateSet.ACTIVE;
+		} else {
+			active = StateSet.INACTIVE;
+		}
+		
+		AtomicBoolean matched = new AtomicBoolean(false);
+		
+		StampCalculator stampCalc = StampCalculatorWithCache
+				.getCalculator(StampCoordinateRecord.make(active, Coordinates.Position.LatestOnMaster()));
+	
+		PatternEntityVersion latestDescriptionPattern = (PatternEntityVersion) Calculators.Stamp.DevelopmentLatest()
+				.latest(TinkarTerm.DESCRIPTION_PATTERN).get();
+        
+		// NOT SURE if this part (forEach loop) is necessary since semanticEntity might not be needed for IT's logic
+		EntityService.get().forEachSemanticForComponentOfPattern(concept.nid(), TinkarTerm.DESCRIPTION_PATTERN.nid(), semanticEntity -> {
+			
+			Latest<SemanticEntityVersion> latest = stampCalc.latest(semanticEntity);
+			UUID semanticEntityUUID = semanticEntity.asUuidArray()[0];
+			
+			if(latest.isPresent())  {
+				matched.set(true);
+			} 
+		});	
+		
+		return matched.get();
+    }
+    
     @Override
     protected boolean assertLine(String[] columns) {
         String loincNum = columns[0]; // LOINC_NUM
         UUID id = uuid(loincNum);
 
         StateSet state = null;
-        if (columns[11].equals("ACTIVE") || columns[11].equals("TRIAL") || columns[11].equals("DISCOURAGED")) {
+        if (columns[11].equals("ACTIVE") || columns[4].equals("TRIAL") || columns[11].equals("DISCOURAGED")) {
```

Comment:
for the "TRIAL" part of the if statement, it is looking at columns[4] instead of columns[11] like the other ones.